marketdata/stream/conn_coder.go

```diff
@@ -24,11 +24,12 @@ func newCoderWebsocketConn(ctx context.Context, u url.URL) (conn, error) {
 	reqHeader := http.Header{}
 	reqHeader.Set("Content-Type", "application/msgpack")
 	reqHeader.Set("User-Agent", alpaca.Version())
-	//nolint:bodyclose // According to its docs: you never need to close resp.Body yourself
-	conn, _, err := websocket.Dial(ctxWithTimeout, u.String(), &websocket.DialOptions{
+	//nolint:bodyclose // Linter Error
+	conn, ignoredResponse, err := websocket.Dial(ctxWithTimeout, u.String(), &websocket.DialOptions{
 		CompressionMode: websocket.CompressionContextTakeover,
 		HTTPHeader:      reqHeader,
 	})
+	alpaca.CloseResp(ignoredResponse)
```

Comment:
`ignoredResponse` is actually `nil` here. I'd just keep it as `_` and add a better explanation to the nolint, e.g.: //nolint:bodyclose // According to the websocket.Dial documentation "you never need to close resp.Body yourself"